.github/ISSUE_TEMPLATE.md

```diff
@@ -6,14 +6,17 @@ A summary of the issue.
 
 ### Steps to Reproduce
 
-1. This is the first step
-2. This is the second step, etc.
+1. (for example) Start a new project with `wagtail start myproject`
+2. Edit models.py as follows...
+3. ...
+
+(It's essential that you provide enough information for someone else to replicate the problem you're seeing. Simply describing something that's broken on your current project is not enough!)
```

Comment:
I think we could remove the parenthesis around this and set this as a paragraph as an intro above the numbered list. And I'm not sure about the 2nd sentence maybe we can rephrase it to something like: a broken functionality can also be a combination of integrating a project with Wagtail.